grpc-interface-support/src/proto.py

```diff
@@ -46,13 +65,14 @@ def get_service_name(self) -> str:
         Returns:
             str: The name of the service.
         """
-        service_name = None
-        with open(self.file_path, encoding="utf-8") as file:
-            for line in file:
-                if line.startswith("service"):
-                    service_name = line.split(" ")[1].strip()
-
-        if service_name is None:
+        if self.__service_name is None:
```

Comment:
do we need this additional check? this is already checked in init...

Reply:
compiler complained :( if I not added this.